src/form-host/wireup.js

```diff
@@ -37,7 +38,6 @@ class FormWireup {
     const loadErrors = form.init();
     if (loadErrors && loadErrors.length) {
       console.log('Load Errors', JSON.stringify(loadErrors));
```

Comment:
can you explain why this is required?  it seems like this is maybe a good "fail-fast" return.

Reply:
an exception sounds like a good idea. i think that's maybe okay that cht-core doesn't